src/proper.erl

```diff
@@ -358,10 +363,10 @@
 -export([get_size/1, global_state_init_size/1,
 	 global_state_init_size_seed/2, report_error/2]).
 -export([pure_check/1, pure_check/2]).
--export([forall/2, implies/2, whenfail/2, trapexit/1, timeout/2]).
+-export([forall/2, implies/2, whenfail/2, trapexit/1, timeout/2, setup/2]).
 
 -export_type([test/0, outer_test/0, counterexample/0, exception/0,
-	      false_positive_mfas/0]).
+	      false_positive_mfas/0, setup_fun/0, finalize_fun/0]).
```

Comment:
It's not so clear why these types are exported; they do not seem to be used (or documented) anywhere...